### PR TITLE
Close the no-use chan in the querycoord node

### PR DESCRIPTION
**Pull request by SimFG**
issue: #20384
Signed-off-by: SimFG <bang.fu@zilliz.com>

### CODE REVIEW

File: internal/querycoordv2/job/scheduler.go [hunk 1]
```diff
@@ -63,6 +63,9 @@ func (scheduler *Scheduler) Stop() {
 	scheduler.stopOnce.Do(func() {
 		close(scheduler.stopCh)
 		scheduler.wg.Wait()
+		for _, queue := range scheduler.queues {
```
Comment (xiaofan-luan) [inline internal/querycoordv2/job/scheduler.go:66]: it should be closed in schedule routine.
Otherwise scheduler might crash

Comment (SimFG) [reply internal/querycoordv2/job/scheduler.go:66]: The `scheduler.wg.Wait()` will make sure that the go-routine has finished.
https://github.com/milvus-io/milvus/blob/c5cd92d36eb9ea35dd7cfd25fa364eaddfd4b1c1/internal/querycoordv2/job/scheduler.go#L120-L143

**Comment by SimFG** [comment]
rerun ut

**Pull request merged**